tests/compare-ref-impl-tests/python/generate_artifacts.py

```diff
@@ -7,16 +7,27 @@
 
 
 class ZserioPackableObject(Protocol):
+    @staticmethod
+    def create_packing_context(
+        zserio_context_node: zserio.array.PackingContextNode,
+    ) -> None:
+        ...
+
+    def init_packing_context(
+        self, zserio_context_node: zserio.array.PackingContextNode
+    ) -> None:
+        ...
+
     def bitsizeof(self, bitposition: int = 0) -> int:
-        pass
+        ...
```

Comment:
Is this valid Python?

Reply:
Yes, see https://docs.python.org/ja/3/library/typing.html .